test/Google.Api.Gax.Grpc.Tests/CallSettingsExtensionsTest.cs

```diff
@@ -282,5 +282,9 @@ public void WithEarlierDeadline_DeadlineIsEarlierThanExistingRetryTotalExpiratio
             Assert.Equal(token, newSettings.CancellationToken);
         }
 
+        [Fact]
+        public void WithExpiration_FIXME()
```

Comment:
Not forgotten - referred to in the PR comment. The plan was to get you to check that I was implementing the appropriate feature, then I'll implement the test :)